corehq/apps/reports/templates/reports/tableau_template.html

```diff
@@ -1,18 +1,11 @@
 
-{% extends "reports/standard/base_template.html" %}
+{% extends "hqwebapp/two_column.html" %}
 {% load compress %}
 {% load hq_shared_tags %}
 {% load i18n %}
-{% block js %}{{ block.super }}
-  {% compress js %}
-    <script src="{% static 'reports/js/report_config_models.js' %}"></script>
-    <script src="{% static "reports/js/base.js" %}"></script>
-    <script src="{% static "reports/js/base_maps.js" %}"></script>
-  {% endcompress %}
-{% endblock %}
-
-{% block filter_panel %}
+{% block js %}{% endblock js %}
```

Comment:
Blanking out this block was good for testing, but for prod, I'd keep it around, since now there's no ancestor js that needs to be eliminated.